helper/builder/localexec/run_and_stream.go

```diff
@@ -0,0 +1,109 @@
+package localexec
```

Comment:
Ah, this is just awesome ! Moving this in its own package was a thing I have wanted to do for a while too !! 👍 👍